lib/players/human.rb

```diff
@@ -0,0 +1,12 @@
+require "pry"
+module Players
+  attr_accessor :board, :game, :player
+  class Human < Player
+    
+    def move(board)
+      puts "Enter move:"
+      user_input = gets.chomp
+      user_input
+    end
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Where would you like to move?"
  gets.chomp
end
```